src/api/db/migrate/20161128115942_add_when_attribute_to_bs_request.rb

```diff
@@ -4,18 +4,13 @@ class TempBsRequest < ActiveRecord::Base
   end
 
   def self.up
-    add_column :bs_requests, :updated_when, :datetime
-    TempBsRequest.find_each do |bs_request|
-      bs_request.updated_when = bs_request.updated_at
-      bs_request.save!
+    ActiveRecord::Base.transaction do
+      add_column :bs_requests, :updated_when, :datetime
+      BsRequest.connection.execute("UPDATE bs_requests AS br SET br.updated_when = br.updated_at")
```

Comment:
in this case definitive. a ruby for loop creating sql statements for each object is much much slower.

I aborted that after > 15 minutes, while the new code is able to do the same within seconds.